tss-esapi/src/structures/tagged/public/rsa.rs

```diff
@@ -251,9 +247,9 @@ impl RsaExponent {
         self.value
     }
 
-    /// Function for checking if a value is valid rsa exponent.
-    pub fn is_valid(value: u32) -> bool {
-        (value > 2 && primal::is_prime(value.into())) || value == 0
+    /// No-op. Does not check whether the value is a valid exponent for RSA.
```

Comment:
Maybe adding `#[deprecated(since="...", note="...")]` would be a good idea for IDEs?